saleor/graphql/order/mutations/draft_order_create.py

```diff
@@ -422,11 +422,6 @@ def save(cls, info: ResolveInfo, instance, cleaned_input):
                         instance, method, manager, update_shipping_discount=False
                     )
 
-            if instance.undiscounted_base_shipping_price_amount is None:
-                instance.undiscounted_base_shipping_price_amount = (
-                    instance.base_shipping_price_amount
-                )
-
```

Comment:
Since `undiscounted_base_shipping_price_amount` value was set through migration and the field has default 0 value in version 3.21 this is not needed anymore